pyracf/common/add_operation_error.py

```diff
@@ -0,0 +1,24 @@
+"""Exception to use when Add operation would alter an existing profile."""
+
+
+class AddOperationError(Exception):
+    """
```

Comment:
I suggest changing the docstring to this instead since it makes it more clear why the exception gets raised.

```python
    """
    Raised when a profile cannot be added because it already exists.
    """
```